lib/src/html_generator.dart

```diff
@@ -5,9 +5,8 @@
 library dartdoc.html_generator;
 
 import 'dart:io';
-
+import 'dart:developer';
```

Comment:
good catch! I just saw the warning that dart:profiler was deprecated.

I'll update the SDK dep.